blog/_posts/2018-06-04-scalac-profiling.md

```diff
@@ -0,0 +1,1326 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jorge Vicente Cantero
+title: "Speeding Up Compilation Time with `scalac-profiling`"
+---
+
+In this blog article, I’d like to introduce you to a new tool that we’ve
+developed at the Scala Center called scalac-profiling, which aims to help you
+better understand what is slowing down compilation in your project.
+
+It turns out that the use of Scala's implicits as well as macros can greatly
+increase compilation times, depending on how they are used and how your
+codebase is organized. Codebases that make use of libraries like Shapeless,
+or which rely on typeclass derivation, may be particularly prone to these
+slow-downs. (As of Scala 2.12, typeclass derivation is based on
+implicitly-triggered macro expansions.)
+
+The goal of this blog post is to help you understand when these things are
+happening in your code, so you can reduce the parts of the code that triggers
+unnecessary implicit searches or macro expansions.
+
+In this blog post, I walk you through how to reduce these compile times with
+scalac-profiling. scalac-profiling is a new Scala Center compiler plugin to
+complement my recent work on the compiler statistics infrastructure merged in
+2.12.5. In this guide, I use the plugin to speed up a module of
+[Bloop](https://scalacenter.github.io/bloop/), a project I've been recently
+working on, to achieve a **8x speedup in compile times**.
```

Comment:
This is kind of confusing. You mean you got an 8x improvement in compile time when compiling bloop, right? Asking because people might quickly read this and think you're talking about how Bloop speeds up peoples' compilation.